src/Common/Utilities/Base64UrlHelper.cs

```diff
@@ -16,7 +16,7 @@
 using System.Globalization;
 using System.Text;
 
-namespace Microsoft.Azure.Commands.Profile.Utilities
+namespace Microsoft.WindowsAzure.Commands.Utilities.Common
```

Comment:
Shall we change the name space?

Reply:
yes, it was incorrect when it was created